### PR TITLE
added toast component

### PR DESCRIPTION
**Pull request by prakashsakari**
Hi kindly review my PR for toast/snackbar. Thank you!

### CODE REVIEW

File: Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    color: var(--text-color-primary);
+    padding: 0.35rem 1rem;
+    font-size: 1rem;
+    border-radius: 4px;
+}
+
+.close {
+    border-radius: 50%;
+    height: 36px;
+    width: 36px;
```
Comment (rohanmathur91) [inline Toast/toast.css]: You can use rem, any specific reason to use px?

**Review by rohanmathur91 — APPROVED** [review]
Added few comment, rest looks good.

File: Toast/toast.css (hunk 1 above)
Comment (prakashsakari) [reply Toast/toast.css]: thanks, updated.

**Review by ianuragsharma — APPROVED** [review]
Looks good to me

**Pull request merged**